src/ng/directive/ngOptions.js

```diff
@@ -661,7 +661,7 @@ var ngOptionsDirective = ['$compile', '$document', '$parse', function($compile,
               listFragment.appendChild(groupElement);
 
               // Update the label on the group element
-              groupElement.label = option.group;
+              groupElement.label = option.group === null ? 'null' : option.group;
```

Comment:
Is this needed for some specific browser? In Chrome that I tried, `null` seems to be converted to `"null"` anyway.